test/aave/Close.test.ts

```diff
@@ -822,4 +822,194 @@ describe(`Strategy | AAVE | Close Position`, async () => {
       expectToBe(new BigNumber(userStEthReserveData.currentATokenBalance.toString()), 'lte', ONE)
     })
   })
+
+  describe(`[1inch] Close Position: With ${tokens.STETH} collateral & ${tokens.ETH} debt`, () => {
+    const multiple = new BigNumber(2)
+    const slippage = new BigNumber(0.2)
+    const USDCPrecision = 6
+    const wBTCPrecision = 8
+    const depositWBTCAmount = amountToWei(new BigNumber(1), wBTCPrecision)
+
+    let openTxStatus: boolean
+    let txStatus: boolean
+
+    let userWBTCReserveData: AAVEReserveData
+    let userUSDCReserveData: AAVEReserveData
+    let system: DeployedSystemInfo
+
+    before(async function () {
+      const shouldRun1InchTests = process.env.RUN_1INCH_TESTS === '1'
+      if (shouldRun1InchTests) {
+        await resetNodeToLatestBlock(provider)
+
+        const { system: _system } = await deploySystem(config, false, false)
+        system = _system
+
+        const addresses = {
+          ...mainnetAddresses,
+          operationExecutor: system.common.operationExecutor.address,
+        }
+
+        const proxy = system.common.dsProxy.address
+        const debtToken = { symbol: tokens.USDC, precision: USDCPrecision }
+        const collateralToken = { symbol: tokens.WBTC, precision: wBTCPrecision }
+
+        await swapUniswapTokens(
+          ADDRESSES.main.WETH,
+          ADDRESSES.main.WBTC,
+          amountToWei(new BigNumber(100)).toFixed(0),
+          ONE.toFixed(0),
+          config.address,
+          config,
+        )
+
+        const COLL_TOKEN = new ethers.Contract(ADDRESSES.main.WBTC, ERC20ABI, provider)
+        await COLL_TOKEN.connect(signer).approve(
+          system.common.userProxyAddress,
+          depositWBTCAmount.toFixed(0),
+        )
+
+        const openPositionTransition = await strategies.aave.open(
+          {
+            depositedByUser: {
+              collateralToken: { amountInBaseUnit: depositWBTCAmount },
+            },
+            positionArgs: {
+              positionId: 123,
+              positionType: 'Multiply',
+              protocol: 'AAVE' as const,
+            },
```

Comment:
we discussed `positionId` already.

But I think `positionType` nad `protocol` can be hardcoded within strategy. The consumer of the library shouldn't be able to set it to anything else while using `aave.open`

Reply:
Yes, positionType is the only big Q. Do we handle within library or pass from FE

Reply:
I currently think the opposite from Damian here - I think we should pass from FE, that way the library is as generic as possible and this same multiply mechanic can be used across both Multiply and Earn positions.  I guess the other way would be to have the same API twice, once for Earn and once for Multiply?  That seems to be more code and more copy-paste that configuring it from the outside.